hack/tools/confdocs.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"github.com/cerbos/cerbos/hack/tools/confdocs"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	interfacePackage = "github.com/cerbos/cerbos/internal/config"
+	interfaceName    = "Section"
+)
+
+func main() {
+	internalDir, err := getAbsToInternalDir()
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+
+	dir, err := filepath.Abs(internalDir)
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+
+	confDoc := confdocs.NewConfDoc(dir, interfaceName, interfacePackage)
+	err = confDoc.Run()
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+}
+
+func getAbsToInternalDir() (string, error) {
+	relative := "."
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+
+	dir := filepath.Join(cwd, relative)
+	if err != nil {
+		return "", err
+	}
+
+	for !strings.HasSuffix(dir, "cerbos") {
+		relative += "/.."
+
+		dir = filepath.Join(cwd, relative)
+		if err != nil {
+			return "", err
+		}
+	}
+
+	dir = filepath.Join(dir, "internal")
+
+	absPath, err := filepath.Abs(dir)
+	if err != nil {
+		return "", err
+	}
+
+	return absPath, nil
+}
```

Comment:
I wonder whether there's a simpler way to do this. One option might be to call [`debug.ReadBuildInfo()`](https://pkg.go.dev/runtime/debug#ReadBuildInfo) and read `Main.Path` to find the module path. (I haven't tried it myself so I could be wrong about it.)

The other way is to find the path to this file (see https://github.com/cerbos/cerbos/blob/aa252b517c94a435a880771f173c762a198b01e5/internal/test/test.go#L59) and  then construct the path from that. We know that this file is in `hack/tools` so the root of the repo is `<this_path>/../..`.

Reply:
I tried using `_, currFile, _, ok := runtime.Caller(0) ` as we used it in tests, and it worked like a charm and it is much shorter now.

Thanks for the suggestion!